cmake/ConfigSGX.cmake

```diff
@@ -30,7 +30,7 @@ endif()
 
 set(SGX_SSL "$ENV{SGX_SSL}")
 if("${SGX_SSL} " STREQUAL " ")
-       set(SGX_SSL /opt/intel/sgxssl)
+    message(FATAL_ERROR "SGX_SSL: undefined environment variable")
```

Comment:
as discussed oob, having defaults only globally in `config.mk` is the right way, so i agree with this change. However, i think we should apply it uniformaly also for the other defaults here; certainly for the ones which are in config.mk (e.g, SGX_MODE and SGX_BUILD) but probably also for SGX_ARCH and SGX_SDK)